proposals/lip-0044.md

```diff
@@ -389,12 +389,12 @@ def registerValidatorWithoutBLSKey(validatorAddress: Address, generatorKey: Publ
 
 #### getValidatorAccount
 
-This function is used to retrieve information about the validator account corresponding to the input `address: Address`. It returns `validatorAccount(address)`. If there is no entry corresponding to `address`, it returns an empty object.
+This function is used to retrieve information about the validator account corresponding to the input `address: Address`. It returns `validatorAccount(address)`. If there is no entry corresponding to `address`, it throws an error.
 
 ```python
 def getValidatorAccount(address: Address) -> ValidatorAccount:
```

Comment:
I checked for the use of this function in other LIPs. I have found only one place where the logic should also change slightly, see here: https://github.com/LiskHQ/lips/blob/e38f4b1307a9a974aadb157e56d2782e9b4d3732/proposals/lip-0050.md?plain=1#L309
I think, we can just remove the check on this line. But possibly we can add try/catch to throw a more specific Exception (saying `'Public key does not correspond to a registered validator.'` instead of `'No validator account found for the input address.'`). I am fine with either option.